01_install_requirements.sh

```diff
@@ -65,3 +60,91 @@ if ! which dep 2>&1 >/dev/null ; then
     mkdir -p $GOPATH/bin
     curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 fi
+
```

Comment:
This should not be merged until a conditional is placed here.  I don't want to do a disconnected install 100% of the time.  If I don't want to do a disconnected install then all the following tasks are unnecessary.  I suggest breaking this all out into a separate file then sourcing it upon met condition.

It's very likely for this pull to be merged so I created a bug for you ... [issues/864](https://github.com/openshift-metal3/dev-scripts/issues/864)

Reply:
Making this conditional on MIRROR_IMAGES sounds reasonable to me, I closed the issue for now as it's confusing to have open bugs against unmerged issues, hopefully we can work this out here before merging.

@yprokule @honza what are your thoughts, can we make these changes conditional, perhaps we could also move the large block of new code into a new file or function to make things more readable?

Reply:
> @yprokule @honza what are your thoughts, can we make these changes conditional, perhaps we could also move the large block of new code into a new file or function to make things more readable?

Moved to a dedicated function in _utils.sh_ Now registry is configured if _MIRROR_IMAGES_ or any other variable matching  pattern _"_LOCAL_IMAGE="_ is set.

@hardys @honza @mazzystr PTAL when U have a moment, thanks!

Reply:
/lgtm

Thanks for that :)